typescript-objects/javascript-objects-quiz-notes.md

```diff
@@ -0,0 +1,50 @@
+# typescript-objects-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What are objects used for?
+  //organizing data, reusability, encapsulation, prototypal inheritance, built-in objects
+- What are object properties?
+  //data about the object, encapsulation within the curly braces
+- Describe object literal notation.
+  //declaring a variable = and putting in object properties within the curly braces
+- How do you remove a property from an object?
+  //using the delete operator and either the dot notation or the bracket notation of what you want to remove
+- What are the two ways to get or update the value of a property?
+  //the dot notation or the bracket notation
+- What is the purpose of interfaces in TypeScript?
+  //it defines the shape of an object, you can use the interface to annotate any objects in the code that match the structure
+- What are two different ways to describe the shape of an object in TypeScript?
+  //embedded within the curly braces, you set the variable and set the typeof you wish for it to take
```

Comment:
In TypeScript, the shape of an object can be described either through interfaces, which define the expected structure of an object, or through type aliases, which can assign a name to any type, including object shapes with specific property types. Both methods allow for strong typing and help in enforcing the structure of objects at compile time.